<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="liuzhanhuideMacBook-Air.local" name="com.okjiaoyu.qa.teacherspace.cases.BaseCase" tests="1" failures="0" timestamp="22 十一月 2018 08:16:37 GMT" time="0.001" errors="1">
  <testcase name="close" time="0.001" classname="com.okjiaoyu.qa.teacherspace.cases.BaseCase">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.okjiaoyu.qa.teacherspace.cases.BaseCase.close(BaseCase.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:350)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.TestNG.privateMain(TestNG.java:1442)
	at com.okjiaoyu.qa.teacherspace.main.MainTest.main(MainTest.java:18)
]]>
    </error>
  </testcase> <!-- close -->
</testsuite> <!-- com.okjiaoyu.qa.teacherspace.cases.BaseCase -->
